pkgs/by-name/ce/celeste64/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  SDL2,
+  libGL,
+  fmodex,
+  libdecor,
+  libxkbcommon,
+  wayland,
+  systemd,
+  alsa-lib,
+  libselinux,
+  withWayland ? true,
+  withSELinux ? false,
+  ...
+}: let
+  rev = "62417823a30582e8561075fe0f7d4b398aedc353";
+  version = builtins.substring 0 6 rev;
+in
+  buildDotnetModule {
+    pname = "celeste64";
+    version = "2024.01.31-${version}";
```

```suggestion
    version = "unstable-2024-01-31";
```

(Or was it date first than "unstable" for unreleased software?)

---

Well actually ... :D 

It should be `version = "0-unstable-2024-01-31";`, as of 3 weeks ago #280312 (no blame on you, I just stumbled on the info)